src/lang/english.txt

```diff
@@ -1465,6 +1467,19 @@ STR_CONFIG_SETTING_TIMEKEEPING_UNITS_HELPTEXT                   :Select the time
 STR_CONFIG_SETTING_TIMEKEEPING_UNITS_CALENDAR                   :Calendar
 STR_CONFIG_SETTING_TIMEKEEPING_UNITS_WALLCLOCK                  :Wallclock
 
+STR_CONFIG_SETTING_MINUTES_PER_YEAR                             :Minutes per year: {STRING2}
+STR_CONFIG_SETTING_MINUTES_PER_YEAR_HELPTEXT                    :Choose the number of minutes in a calendar year. The default is 12 minutes. Set to 0 to freeze calendar time completely. This setting does not affect the economic simulation of the game, and is only available when using real-time units.
+
+# Value strings used in Settings GUI
+STR_CONFIG_SETTING_MINUTES_PER_YEAR_VALUE                       :{NUM} minutes
+###setting-zero-is-special
+STR_CONFIG_SETTING_MINUTES_PER_YEAR_FROZEN                      :Frozen
+
+# Value strings used in Generate World GUI
+STR_CONFIG_SETTINGS_MINUTES_PER_YEAR_VALUE_FROZEN               :{BLACK}Frozen
+STR_CONFIG_SETTINGS_MINUTES_PER_YEAR_VALUE_PERCENT              :{BLACK}{NUM} minute{P 0 "" s} ({NUM}% game speed)
+STR_CONFIG_SETTINGS_MINUTES_PER_YEAR_VALUE_ONE_PERCENT          :{BLACK}{NUM} minute{P 0 "" s} (<1% game speed)
+
```

Comment:
I think you forgot to remove some strings? :)

Reply:
Nevermind, GitHub interface playing tricks on me.

Guess the next question is: weren't you going to drop the %? :)

Reply:
Yes, I haven't gotten to it yet. 😉